dot/node_integration_test.go

```diff
@@ -75,6 +75,15 @@ func TestNewNode(t *testing.T) {
 	stateConfig := state.Config{
 		Path:     initConfig.BasePath,
 		LogLevel: logLevel,
+		GenesisBABEConfig: &types.BabeConfiguration{
```

Comment:
maybe just have some predefined test variable that represents this default test config.